WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -0,0 +1,82 @@
+require 'i18n/tasks/scanners/file_scanner'
+class WCAFileScanner < I18n::Tasks::Scanners::FileScanner
+  include I18n::Tasks::Scanners::RelativeKeys
+  include I18n::Tasks::Scanners::Results
+
+  def active_record?(model)
+    # FIXME: Find a way to determine this dynamically ?
+    model != "contact"
+  end
+
+  def extract_model_attr(key)
+    model_key = key.gsub(/^(\w+)\..*$/, '\1').singularize
+    # Some dirty tricks to assign the detected field to the "correct" model
+    # For most controllers the forms modify a model which has the same name
+    # as the controller, however for some of them they modified a single different
+    # model (eg: the `admin` forms modify `person`s, and only that model).
+    model_name = case model_key
+                 when "devise" then "user"
+                 when "admin" then "person"
+                 when "oauth" then "doorkeeper/application"
+                 else model_key
+                 end
+    attribute = key.gsub(/.*\.(\w+)$/, '\1')
+    [model_name, attribute]
+  end
+
+  def occurrence_from_line(path, line, line_number, position)
+    Occurrence.new(
+      path:     path,
+      pos:      line_number + position,
+      line_num: line_number,
+      line_pos: position,
+      line:     line,
+      raw_key: nil)
+  end
+
+  # @return [Array<[absolute key, Results::Occurrence]>]
+  def scan_file(path)
+    text = read_file(path)
+    retval = []
+    line_number = 0
+    text.each_line do |line|
+      line_number += 1
+      # Scan for possible radio choices
+      if line =~ /^\s*<%= f.input :(\w+), as: :radio_buttons, collection: \[(.*)\](.*)%>/
```

Comment:
What if we split attributes in multiple lines for readability, like:
```ruby
f.input :something, as: :radio_buttons,
                 collection: [....]
```
Also automaticly generated collection arrays doesn't match but there's no way to do so I guess.